server/src/main/java/org/opensearch/common/Strings.java

```diff
@@ -66,7 +66,7 @@
  */
 public class Strings {
 
-    public static final String[] EMPTY_ARRAY = new String[0];
+    public static final String[] EMPTY_ARRAY = org.opensearch.core.common.Strings.EMPTY_ARRAY;
```

Comment:
Why do we need to copy that and not use `org.opensearch.core.common.Strings.EMPTY_ARRAY` everyhwere?

Reply:
Minimizing the surface area of the change. There are still 72 usages of o.o.common.Strings in classes that do not take a dependency on o.o.core.common.Strings. I'm trying to minimize explicit class litter in this PR. I think there's already enough as it is.